maker/src/main.rs

```diff
@@ -39,7 +39,11 @@ async fn main() -> Result<()> {
 
     let (_, quote_receiver) = bitmex::subscribe()?;
 
-    let mission_success = rocket::build()
+    let figment = rocket::Config::figment()
+        .merge(("address", "0.0.0.0"))
```

Comment:
perhaps a comment stating that we need this for Docker would be useful - might be useful for next projects :)

Reply:
I 'll overwrite this when doing  #217 and let the user pass it in.